guide/getting-started/hardware.md

```diff
@@ -0,0 +1,96 @@
+
+## Recovery phrase storage
+
+Simple, cheap and robust storage solutions for recovery phrases come in modest forms. [Seedplate](https://bitcoinseedbackup.com/){:target="_blank"}, for example, is a metal plate in which users can etch the words of their recovery phrase for long-term offline storage. [CardWallet](https://www.cardwallet.com){:target="_blank"}, in comparison, is the size of a credit card and comes pre-loaded with a recovery phrase (hidden behind a security seal) and a matching bitcoin address. And if you like an even simpler solution, a piece of paper will do. Although paper is not as durable and may be easy to loose, it is one of the strengths of Bitcoin that such simple solutions are possible.
+
+{% include image.html
+   image = "/assets/images/guide/getting-started/hardware/hardware-seed-storage.jpg"
+   retina = "/assets/images/guide/getting-started/hardware/hardware-seed-storage@2x.jpg"
+   alt-text = "Examples of seed storage options"
+   width = 800
+   height = 400
+%}
+
+## Hardware wallets
+
+You may already be familar with key fobs from your bank or work. They are small devices with specific built-in authentication that is used to control and secure access to a computer system. Key fobs are not connected to the internet or any other network, so they cannot easily be hacked without physical access.
+
+{% include image.html
+   image = "/assets/images/guide/getting-started/hardware/hardware-wallets.jpg"
+   retina = "/assets/images/guide/getting-started/hardware/hardware-wallets@2x.jpg"
+   alt-text = "Examples of hardware wallets"
+   caption = '<a href="https://coldcardwallet.com" target="_blank">Coldcard</a>, <a href="https://www.ledger.com/" target="_blank">Ledger</a>, <a href="https://trezor.io" target="_blank">Trezor</a>, and <a href="https://foundationdevices.com" target="_blank">Foundation</a>'
+   width = 800
+   height = 400
+%}
+
+Hardware wallets for bitcoin play a similar role but with several differences. Similar to key fobs, they securely store an authentication mechanism and isolate it from other devices and networks. In the case of bitcoin, these are the seed, associated keys and all functions that require direct interaction with them.
+
+Because hardware wallets are so optimized for this specific uses case, they have very small screens and limited input options. Most interactions with them happen via desktop software that is also provided by the manufacturer. So users perform most wallet interactions on their desktops, and the application sends only specific requests to the hardware wallet to verify or sign data. This provides a great balance of security and convenience, and keeps hardware wallets generally affordable for most use cases.
```

Comment:
Because hardware wallets are so optimized for this specific use case... (not uses case)

Reply:
Fixed. Thank you.